ext/libgecode3/extconf.rb

```diff
@@ -112,7 +112,13 @@ def self.num_processors
   def self.run_build_commands
     setup_env
     patch_configure
-    system(*configure_cmd) &&
+    system("libtoolize --force") &&
+      system("aclocal") &&
+      system("autoheader") &&
+      system("automake --force-missing --add-missing || true") &&
```

Comment:
Just to add a note here. This command will fail due to other issues but it will still create the files we need.